repository_creation/create_repository.go

```diff
@@ -237,7 +282,9 @@ func createRepo(ctx context.Context, client *github.Client, repo *github.Reposit
 }
 
 // getConfigurationsForNewRepo gets required configuration information from the end user
-func getConfigurationsForNewRepo(name, description string, projType project_generation.ProjectType, personalAccessToken string, branchStrategy string) (accessToken, repoName, repoDescription, defaultBranch string) {
+func getConfigurationsForNewRepo(name, description string, projType project_generation.ProjectType, personalAccessToken, branchStrategy, teamSlugsInput string) (accessToken, repoName, repoDescription, defaultBranch, repoTeamSlugsInput string) {
+	ctx := context.Background()
+
```

Comment:
When I tried this out it gave me the message on line 294, which includes a link for creating a personal access token, because my personal access token had expired. When I clicked the link in the message it gave me a 404 with the message "Ooops! It looks like this page doesn't exist." It looks like the link ends in ): when it shouldn't do, so it be good to correct that I think.